src/content/advanced/configure-dex-in-your-cluster/index.md

```diff
@@ -69,6 +69,25 @@ spec:
       issuerURL: https://dex.<CLUSTER>.<BASEDOMAIN>
 ```
 
+{{< /tab >}}
+{{< tab title="Openstack">}}
+
+```yaml
+apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+kind: KubeadmControlPlaneTemplate
+spec:
+  template:
+    spec:
+      kubeadmConfigSpec:
+        clusterConfiguration:
+          apiServer:
+            extraArgs:
+              oidc-issuer-url: https://dex.<CLUSTER>.<BASEDOMAIN>
+              oidc-client-id: dex-k8s-authenticator
+              oidc-username-claim: email
+              oidc-groups-claim: groups
+```
```

Comment:
Shouldn't we document this instead? https://github.com/giantswarm/cluster-openstack/blob/5a94e4ad6668cd9532882a3023c14dda7c1456ab/helm/cluster-openstack/values.yaml#L13-L14

Reply:
Good point. We want `kubectl-gs`/`cluster-openstack` to be the interface through which customers manage clusters for now. I can update it.

Reply:
But you mean generate the resources with kgs or use boolean value to set it up?
IMO we need to give users ability to set different OIDC endpoint and no limit us

Reply:
Also the bool flag has dissapear instead of the settings

Reply:
Customers create clusters using the `cluster-openstack` helm chart so we should instead show these values https://github.com/giantswarm/cluster-openstack/blob/89b33cfbb78a7d9092dbe5e953aa528447edc361/helm/cluster-openstack/values.yaml#L16-L21 or the corresponding `kubectl-gs` flags https://github.com/giantswarm/kubectl-gs/blob/2836cded90bb4ee9c11809cd00f10490dff71710/cmd/template/cluster/flag.go#L57-L61 in the docs.

Reply:
ok then no need to add `enabled` anymore, right?

Reply:
Let's go with this and I will do a follow up PR with the kgs flags and helm chart in different section than legacy providers

Reply:
This matches how we display the required changes for other providers. It will take more work to show the old method and the new method (using `kubectl-gs` flags).